onnxruntime/test/providers/nnapi/nnapi_basic_test.cc

```diff
@@ -394,6 +394,19 @@ TEST(NnapiExecutionProviderTest, TestQDQSoftMax) {
                   });
 }
 
+TEST(NnapiExecutionProviderTest, TestQDQConcat) {
```

Comment:
Do we have any tests that validate we don't use NNAPI if the input/output types or zp/scale values are unacceptable that validate those paths work correctly? 

Possibly this applies to other tests here and we should flesh out the negative tests (i.e. NNAPI wasn't able to be used for some reason) next week.

Reply:
So far we don't have tests for that, but this can be easily added

Reply:
Added Task 1677444: Add more NNAPI QDQ tests to check failure paths.